StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -166,6 +166,10 @@ extension PaymentSheet {
         // MARK: Internal
         internal var linkPaymentMethodsOnly: Bool = false
 
+        /// If enabled, PaymentSheet will offer V2 Link features, such as passthrough mode
+        /// 🚧 Under construction, do not use 🚧
+        @_spi(STP) public var allowLinkV2Features: Bool = false
```

Comment:
Instead of STP, add some cautionary verbage to the _spi, e.g.: `@_spi(STP_UNDER_CONSTRUCTION)` ?

Reply:
This is (and always will be) internal-only, so it should be scoped the same as our other internal imports. I'm deleting this flag at the end. (In MOBILESDK-1468)

Reply:
I'll remove the "under construction" comment to avoid confusion